tests/ui/abi/unsupported-varargs-fnptr.rs

```diff
@@ -1,5 +1,6 @@
 // FIXME(workingjubilee): add revisions and generalize to other platform-specific varargs ABIs,
 // preferably after the only-arch directive is enhanced with an "or pattern" syntax
+// NOTE: This deliberately tests an ABI that supports varargs, so no `extern "rust-invalid"`
```

Comment:
An alternative would be to, in fact, say "yes, `extern "rust-invalid"` can support varargs", but that seems silly.